vms/job/forms.py

```diff
@@ -35,3 +35,4 @@ class SearchJobForm(forms.Form):
     country = forms.RegexField(
         regex=r'^[(A-Z)|(a-z)|(\s)|(\-)]+$', max_length=75, required=False)
     event = forms.CharField(required=False)
+
```

Comment:
shouldn't this change be in previous commit if you are including this change

Reply:
@anjali-dhanuka please update these changes.